css/services.css

```diff
@@ -0,0 +1,277 @@
+    padding-bottom: 6px;
+}
+
+#centerSection {
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: space-between;
+    margin-top: -9px;
+    margin-bottom: -20px;
+}
+
+
+#centerSection div {
+    width: 47.7%;
+    display: flex;
+    height: auto;
+    flex-direction: column;
+    background-color: rgb(211, 211, 211);
+    border: 1px solid black;
+    padding: 15px 2.5%;
+    margin: 15px 0% 27px;
+    justify-content: space-around;
+}
+
+#centerSection div h4 {
+    margin-bottom: 5px;
+}
+
+#centerSection div p {
+    line-height: 1.3
+}
+
+.learnMore {
+    font-size: 1.2rem;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    border: 1px solid black;
+    border-radius: 4px;
+    background-color: rgb(231, 231, 231);
+    width: 37%;
+    height: 34px;
+    margin-top: 10px;
+}
+
+#bottomSection {
+    display: flex;
+    width: 100%;
+    justify-content: space-between;
+    margin-bottom: 92px;
+}
+
+#bottomSection div {
+    display: flex;
+    flex-direction: column;
+    align-items: space-between;
+    justify-content: space-between;
+    height: 270px;
+    width: 45.4%;
+}
+
+#bottomSection div h4 {
+    margin-top: 3px;
+    margin-bottom: 5px;
+}
+
+ul {
+    list-style: square inside;
+    display: flex;
+    flex-direction: column;
+    height: 40%;
+    justify-content: space-between;
+}
+
+footer {
+    display: flex;
+    justify-content: center;
+    height: 50px;
+}
+
+@media (max-width: 800px) {
+    .container {
+        max-width: 705px;
+    }
+    header {
+        flex-direction: column-reverse;
+        align-items: center;
+        margin-top: 3px;
+        height: 145px;
+    }
+    nav {
+        width: 100%;
+        justify-content: space-around;
+        margin-right: 5.1%;
+        padding-left: 1.2%;
+    }
+    .divisionLine {
+        margin: 35px auto 0px;
+        width: 100%;
+    }
+    #centerSection {
+        flex-direction: column;
+    }
+    #centerSection div {
+        width: 100%;
+        height: auto;
+    }
+    .services-info-img {
+        display: none;
+    }
+    #bottomSection div {
+        width: 100%;
+    }
+    #bottomSection {
+        margin-bottom: 40px;
+    }
+}
+
+@media (max-width: 500px) {
+    .container {
+        max-width: 435px;
+    }
+    header {
+        width: 100%;
+        margin-top: -5px;
+        height: initial;
+    }
+    nav {
+        flex-direction: column;
+        width: 100%;
+        height: 386px;
+        font-size: 2rem;
+        padding-top: 18px;
+        padding-left: 8%;
+    }
+    nav a {
+        width:100%;
+        text-align: center;
+        padding-top: 17px;
+        border-top: 1px solid black;
+    }
+    nav a:last-child{
+        padding-bottom: 20px;
+        border-bottom: 1px solid black;
+    }
+    .divisionLine {
+        width: 92%;
+    }
+    #bottomSection div {
+        height: auto;
+    }
+    #bottomSection div ul {
+        text-align: center;
+        line-height: 1.3;
```

Comment:
I love that you went for the stretch goals! However, this section isn't fully aligned.